priv/static/phoenix.js.map

```diff
@@ -0,0 +1 @@
InMs       = 5000\n    this.endPoint        = null\n    this.token           = null\n    this.sig             = null\n    this.skipHeartbeat   = true\n    this.onopen          = function(){} // noop\n    this.onerror         = function(){} // noop\n    this.onmessage       = function(){} // noop\n    this.onclose         = function(){} // noop\n    this.states          = SOCKET_STATES\n    this.upgradeEndpoint = this.normalizeEndpoint(endPoint)\n    this.pollEndpoint    = this.upgradeEndpoint + (/\\/$/.test(endPoint) ? \"poll\" : \"/poll\")\n    this.readyState      = this.states.connecting\n\n    this.poll()\n  }\n\n  normalizeEndpoint(endPoint){\n    return endPoint.replace(\"ws://\", \"http://\").replace(\"wss://\", \"https://\")\n  }\n\n  endpointURL(){\n    return this.pollEndpoint + `?token=${encodeURIComponent(this.token)}&sig=${encodeURIComponent(this.sig)}`\n  }\n\n  closeAndRetry(){\n    this.close()\n    this.readyState = this.states.connecting\n  }\n\n  ontimeout(){\n    this.onerror(\"timeout\")\n    this.closeAndRetry()\n  }\n\n  poll(){\n    if(!(this.readyState === this.states.open || this.readyState === this.states.connecting)){ return }\n\n    Ajax.request(\"GET\", this.endpointURL(), \"application/json\", null, this.ontimeout.bind(this), (status, resp) => {\n      if(resp && resp !== \"\"){\n        var {token, sig, messages} = JSON.parse(resp)\n        this.token = token\n        this.sig = sig\n      }\n      switch(status){\n        case 200:\n          messages.forEach( msg =>  this.onmessage({data: JSON.stringify(msg)}) )\n          this.poll()\n          break\n        case 204:\n          this.poll()\n          break\n        case 410:\n          this.readyState = this.states.open\n          this.onopen()\n          this.poll()\n          break\n        case 0:\n        case 500:\n          this.onerror()\n          this.closeAndRetry()\n          break\n        default: throw(`unhandled poll status ${status}`)\n      }\n    })\n  }\n\n  send(body){\n    Ajax.request(\"POST\", this.endpointURL(), \"application/json\", body, this.onerror.bind(this, \"timeout\"), (status, resp) => {\n      if(status !== 200){ this.onerror(status) }\n    })\n  }\n\n  close(code, reason){\n    this.readyState = this.states.closed\n    this.onclose()\n  }\n}\n\n\nexport var Ajax = {\n\n  states: {complete: 4},\n\n  request: function(method, endPoint, accept, body, ontimeout, callback){\n    let req = XMLHttpRequest ? new XMLHttpRequest() : // IE7+, Firefox, Chrome, Opera, Safari\n                               new ActiveXObject(\"Microsoft.XMLHTTP\") // IE6, IE5\n    req.open(method, endPoint, true)\n    req.setRequestHeader(\"Content-type\", accept)\n    req.onerror = function(){ callback && callback(500, null) }\n    req.onreadystatechange = () => {\n      if(req.readyState === this.states.complete && callback){ callback(req.status, req.responseText) }\n    }\n    if(ontimeout){ req.ontimeout = ontimeout }\n\n    req.send(body)\n  }\n}\n"]}
```

Comment:
Do we need this .map file? Shouldn't the compiler phoenix.js already be minified? I.e. compiled in production mode?